components/cloud-foundry/src/view/dashboard/cluster/organization/space/detail/summary-tile/space-summary-tile.directive.js

```diff
@@ -133,7 +135,7 @@
       }
     }, function (roles) {
       // Present the user's roles
```

Comment:
This is another fix which I snuck in but mentioned in the description. Fixes the space role labels in the space summary tile (we had a similar issue for the org summary tile)